### PR TITLE
Become

### PR DESCRIPTION
**Pull request by fengor**
seperate pull request for sudo -> become change.

Upped requirements.txt to ansible 1.9.4

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-ansible==1.6.6
+ansible==1.9.4
```
Comment (neuhaus) [inline requirements.txt]: why 1.9.4?

**Comment by fengor** [comment]
because become is definately available in 1.9.4

**Comment by ariddell** [comment]
The version string should be ``ansible>=1.9.4,<2.0``, IMHO. Could this and #487 be reconciled somehow? The PRs are not very clear. From the PR text and commit messages I'm not sure what the problem is and/or why the commits solve the problem.

**Comment by ariddell** [comment]
What about a single PR that deals with upgrading to Ansible to 2.0+ and deals with the ansible-lint issue in #487. It should have a title like "Upgrade Ansible to 2.0". I'm willing to do this if nobody else gets around to it.

**Comment by neuhaus** [comment]
We don't know what version of ansible the users have. Ansible (often) runs on a different machine than the one that is being configured. If it's a Ubuntu trusty box they probaby have Ansible 1.5.4. Debian jessie ships with Ansible 1.7.2

**Comment by carljm** [comment]
There is no reason to rely on a system-installed Andible version; a project like sovereign needs to be able to pin itself to a specific Ansible version and rely on that. The instructions should include creating a Python virtualenv and running `pip install -r requirements.txt` within it, so we always have exactly the specified Ansible version available.

**Comment by ariddell** [comment]
I agree. Let's just decide on a version to support and have the
requirements.txt file reflect this. I believe Ansible is using semantic
versioning so >=2.0,<2.1 sounds good to me.

**Comment by nodiscc** [comment]
For what it's worth many users don't want to depend on pip installs for production/trusted machines. I prefer sticking to the ansible version provided by my package manager. As your current target is Jessie, I think you should also expect ansible to be at the version provided in Jessie (even if the admin/deployement machine is not always using the same OS/version).

Just my 2 cents.

**Comment by ariddell** [comment]
IIRC, we already required a version higher than the version installed by default with wheezy/trusty.

Remember, we're talking about the local copy of ansible. Having a virtualenv or using ``pip install --user`` is no big deal.

**Comment by ariddell** [comment]
Since master is doing fine could we apply this just to jessie? We could upgrade ansible to 2.0 and ansible-lint as well.

**Comment by ariddell** [comment]
Fixed with #508.

**Pull request closed without merging**